packages/venia-ui/lib/components/CheckoutPage/ItemsReview/itemsReview.css

```
@@ -1,12 +1,60 @@
-.text_content {
-    padding: 25px;
-    padding-top: 50px;
+.total_quantity {
     font-weight: bold;
 }
 
-.container {
-    min-height: 20rem;
+.show_all_items_button {
+    composes: root_normalPriority from '../../Button/button.css';
+    width: 100%;
+    border: none;
+}
+
+.show_all_items_button > span {
+    font-weight: bold;
+    font-size: 0.9rem;
+}
+
+.show_all_footer_container {
+    border-top: 1px solid rgb(var(--venia-border));
+    padding: 0.5rem 0rem;
+}
+
+.items_review_container {
+    min-height: 2rem;
     border: 1px solid rgb(var(--venia-border));
     border-radius: 5px;
+}
+
+.items_container {
+    display: grid;
+    gap: 2rem;
     padding: 1.5rem 1rem;
 }
+
+.item_root {
+    display: grid;
+    grid-template-columns: 1fr 4fr;
```

Layouts made up of entirely flexible tracks do exist, but they're very unusual. This isn't one of them. The image has a fixed width and doesn't have much use for extra space, so all the extra space should go to the second column.

```suggestion
    grid-template-columns: auto 1fr;
```

---

Nice one. I was not aware of this 👍